editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -258,12 +259,7 @@ const InviteToPayForm: React.FC<InviteToPayFormProps> = ({
       >
         {"I want to pay for this application myself"}
       </Button>
-      <Typography variant="body2">or</Typography>
-      <Link component="button">
-        <Typography variant="body2">
-          Save and return to this application later
-        </Typography>
-      </Link>
+      <SaveResumeButton />
```

Comment:
Maybe one way to address the bug of currently showing on _all_ services is to check if the current route has a `?sessionId=` param, and if it does conditionally show the `<SaveResumeButton />`? That feels a little hacky, but far less complicated/tricky than getting into `handleSubmit`/`Card` props?